cloud/ec2_util.go

```diff
@@ -28,13 +28,17 @@ import (
 const (
 	EC2ErrorNotFound        = "InvalidInstanceID.NotFound"
 	EC2DuplicateKeyPair     = "InvalidKeyPair.Duplicate"
+	EC2TemplateNameExists   = "InvalidLaunchTemplateName.AlreadyExistsException"
 	EC2InsufficientCapacity = "InsufficientInstanceCapacity"
 	EC2InvalidParam         = "InvalidParameterValue"
 	EC2VolumeNotFound       = "InvalidVolume.NotFound"
 	EC2VolumeResizeRate     = "VolumeModificationRateExceeded"
 )
 
-var EC2InsufficientCapacityError = errors.New(EC2InsufficientCapacity)
+var (
+	EC2InsufficientCapacityError = errors.New(EC2InsufficientCapacity)
+	EC2TemplateNameExistsError   = errors.New(EC2TemplateNameExists)
```

Comment:
Does this need to be exported? Seems like it's never used in any other package.

Reply:
done